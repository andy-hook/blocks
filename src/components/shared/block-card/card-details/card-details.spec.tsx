import React from "react"
import CardDetails from "./card-details"
import { renderWithTheme } from "@test-utils"

describe("<CardDetails />", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme("dark", <CardDetails />)
    expect(tree).toMatchSnapshot()
  })
})
