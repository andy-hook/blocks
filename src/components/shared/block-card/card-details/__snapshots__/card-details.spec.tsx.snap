// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardDetails /> renders correctly 1`] = `
<div
  className="sc-htpNat kqEEqD"
>
  <div
    className="sc-EHOje ffvkok"
  >
    <div
      className="sc-bdVaJa xGfmu sc-bxivhb hIEsEO"
      size="md"
    >
      Size
    </div>
    <div
      className="sc-bdVaJa jmbRgh sc-ifAKCX dHHSVa"
      size="md"
    />
  </div>
  <div
    className="sc-bZQynM hNVbvw"
  >
    <div
      className="sc-bdVaJa xGfmu sc-bxivhb hIEsEO"
      size="md"
    >
      Difficulty
    </div>
    <div
      className="sc-bdVaJa jmbRgh sc-ifAKCX dHHSVa"
      size="md"
    />
  </div>
  <div
    className="sc-gzVnrw jnrzUa"
  >
    <div
      className="sc-bdVaJa xGfmu sc-bxivhb hIEsEO"
      size="md"
    >
      Miner
    </div>
    <div
      className="sc-bdVaJa jmbRgh sc-ifAKCX dHHSVa"
      size="md"
    />
  </div>
</div>
`;
